dojo/db_migrations/0032_remove_finding_duplicate_list.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.9 on 2020-02-14 09:14
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0031_finding_component'),
```

Comment:
isn't the same as recently added `component_name` ?

Reply:
it's depending on that migration yes